### PR TITLE
cgroup_fuse: actually make asz check mean something

### PR DESCRIPTION
**Pull request by brauner**
Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>

### CODE REVIEW

File: tests/main.sh [hunk 1]
```diff
@@ -79,6 +79,8 @@ RUNTEST() {
 	fi
 }
 
+TESTCASE"Recursive Walk"
+for i in `seq 1 100`; ls -al "${LXCFSDIR}"; done
```
Comment (stgraber) [inline tests/main.sh]: Bit of a lie as it's not actually recursive. Also, we like our tests to be quiet unless they fail.

Comment (brauner) [reply tests/main.sh]: Ah, forgot the R

Comment (brauner) [reply tests/main.sh]: fixed

Comment (stgraber) [reply tests/main.sh]: not seeing a fix so either you're still working on the test or you forgot to push :)

File: tests/main.sh [hunk 2]
```diff
@@ -79,6 +79,8 @@ RUNTEST() {
 	fi
 }
 
+TESTCASE"Recursive Walk"
+for i in `seq 1 100`; ls -alR "${LXCFSDIR}" &> /dev/null; done
```
Comment (stgraber) [inline tests/main.sh]: `&>` is a bashism, the non-bash variant is `>/dev/null 2>&1`

Comment (brauner) [reply tests/main.sh]: thanks!

**Comment by stgraber** [comment]
jenkins: test this please

**Comment by brauner** [comment]
> jenkins: test this please

That's LXCFS what would that do? :)

**Comment by brauner** [comment]
Ah, the DCO stuff

File: tests/main.sh [hunk 3]
```diff
@@ -79,6 +79,8 @@ RUNTEST() {
 	fi
 }
 
+TESTCASE="Recursive Walk"
+for i in `seq 1 100`;do ls -alR "${LXCFSDIR}" >/dev/null 2>&1; done
```
Comment (stgraber) [inline tests/main.sh]: Actually, we very much want to see the errors, so remove the `2>&1` part

File: tests/test_readdir [hunk 4]
```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```
Comment (stgraber) [inline tests/test_readdir]: Need to make the file executable

Comment (brauner) [reply tests/test_readdir]: Sigh, sorry!

**Pull request merged**